src/group_theory/order_of_element.lean

```diff
@@ -303,6 +303,29 @@ lemma is_cyclic_of_order_of_eq_card [group α] [decidable_eq α] [fintype α]
   (set.subset_univ _)
   (by {rw [fintype.card_congr (equiv.set.univ α), ← hx, order_eq_card_gpowers], refl})⟩⟩
 
+lemma is_cyclic_of_prime_card [group α] [fintype α] (p : ℕ) (hp : nat.prime p) (h : fintype.card α = p) : is_cyclic α :=
```

Comment:
Not sure if what's more standard, but `order_of_eq_prime` in this file uses hypotheses like this:
```suggestion
lemma is_cyclic_of_prime_card [group α] [fintype α] {p : ℕ} [fact p.prime] (h : fintype.card α = p) : is_cyclic α :=
```
If `fact` is undesirable here, `p : ℕ` can still be implicit, right?

Reply:
`p` should definitely be implicit. Whether or not to use `fact` should depend on related lemmas/where this one will be applied. I think there's no strict rule, but it should be consistent with anything similar.